packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -499,7 +499,7 @@ export default defineComponent({
 
 			this.credentialData = {
 				...this.credentialData,
-				scopes,
+				scopes: scopes as unknown as CredentialInformation,
```

Comment:
could we define those better in the original type?

Reply:
Updated.